### ISSUE TITLE
Utils

### ISSUE THREAD

**Issue by Jamerson1793** [trevorglick/mythic_phone#3]
Is their a Download for the Resource Utils? located anywhere, it seems to be calling lots of exports from that resource.

**Comment by trevorglick** [comment]
No, that resource is just a utility resource I created to get phone_number
added to the esx.getplayerbyid() method.  I just updated that method to
return that from the sql query.
